x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/action/EqlActionIT.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.eql.action;
+
+import com.carrotsearch.randomizedtesting.annotations.ParametersFactory;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.elasticsearch.Build;
+import org.elasticsearch.action.bulk.BulkRequestBuilder;
+import org.elasticsearch.action.bulk.BulkResponse;
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.action.support.WriteRequest;
+import org.elasticsearch.common.xcontent.XContentType;
+import org.elasticsearch.search.SearchHit;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import static org.hamcrest.Matchers.equalTo;
+
+public class EqlActionIT extends AbstractEqlIntegTestCase {
+
+    static final String indexPrefix = "endgame";
+    static final String testIndexName = indexPrefix + "-1.4.0";
+    protected static final String PARAM_FORMATTING = "%1$s.test";
+
+
+    @BeforeClass
+    public static void checkForSnapshot() {
+        assumeTrue("Only works on snapshot builds for now", Build.CURRENT.isSnapshot());
```

Comment:
A previous comment from @imotov in the original PR - https://github.com/elastic/elasticsearch/pull/52248/files#r379168077 - was ignored. I'm curious what was the conclusion back then regarding this line.

Reply:
Missed this one. We still have the same check for EqlRestIT as well. No harm during development of this feature. Can be removed later when the feature is ready for release. @imotov let me know if you have a strong opinion on this.